docker-compose.yml

```diff
@@ -101,7 +101,7 @@ mapfishapp:
   image: georchestra/mapfishapp:latest
   volumes:
     - /etc/georchestra:/etc/georchestra
-    - mapfishapp_uploads:/var/local/uploads
+    - mapfishapp_uploads:/mnt/mapfishapp_uploads
```

Comment:
We need to update this : https://github.com/georchestra/georchestra/blob/master/mapfishapp/src/docker/Dockerfile#L15
And : https://github.com/georchestra/georchestra/blob/master/mapfishapp/src/docker/docker-entrypoint.d/00-adjust-perms#L3